k8s/openebs-operator.yaml

```diff
@@ -609,6 +609,10 @@ webhooks:
         apiGroups: ["*"]
         apiVersions: ["*"]
         resources: ["persistentvolumeclaims"]
+      - operations: [ "CREATE", "UPDATE", "DELETE" ]
```

Comment:
this might now need change as 'DELETE' is not handled as of now

Reply:
yes. I will do that.